dr-multi-replica.md

```diff
@@ -0,0 +1,211 @@
+---
+title: DR Solution based on Multiple Replicas in a Single Cluster
+summary: Learn about the multi-replica disaster recovery solution for a single cluster.
+---
+
+# DR Solution based on Multiple Replicas in a Single Cluster
+
+This document describes the disaster recovery (DR) solution based on multiple replicas in a single cluster. The document is organized as follows:
+
+- Solution introduction
+- How to set up a cluster and configure replicas
+- How to monitor the cluster
+- How to perform a DR switchover
+
+## Introduction
+
+Important production systems usually require regional DR with zero RPO and minute-level RTO. A Raft-based distributed database, TiDB provides multiple replicas, which allows it to support regional DR with data consistency and high availability guaranteed. Considering the small network latency between available zones (AZs) in the same region, we can dispatch business traffic to two AZs on the same region simultaneously, and achieve load balance among AZs on the same region by properly locating the Region Leader and PD Leader.
+
+> **Note:**
+>
+> The term "Region" in TiKV means a range of data while the term "region" means a physical location. The two terms are not interchangeable.
+
+## Set up a cluster and configure replicas
```

Comment:
This is for TiUP Cluster deployments. What about TiDB Operator based deployments?

Reply:
@allengaoo PTAL

Reply:
currently, we do not have a document for TiDB operator based deployments. Please file a FRM request if this is needed.